docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+RAPIDS plugin can remarkably accelerate the computing part of the whole SQL query by
+leveraging GPUs, but it’s hard to accelerate the data reading process when the data
+is in the cloud file system because of network overhead. Alluxio is an open source
```

Comment:
```suggestion
The RAPIDS plugin can remarkably accelerate the computing part of a SQL query by
leveraging GPUs, but it’s hard to accelerate the data reading process when the data
is in a cloud file system because of network overhead. Alluxio is an open source
```